### PR TITLE
imx: drop imx_warm_reset workaround

### PR DESCRIPTION
**Pull request by igoropaniuk**
Drop imx_warm_reset command in favor of "reset -w", which
also triggers warm reset.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>

Please do not submit a Pull Request via github.  Our project makes use of
mailing lists for patch submission and review.  For more details please
see https://www.denx.de/wiki/U-Boot/Patches

### CODE REVIEW

**Comment by mike-scott** [comment]
Requires https://github.com/foundriesio/meta-lmp/pull/352

**Review by Tim-Anderson — APPROVED** [review]
LGTM must be after meta-lmp PR #352

**Comment by ricardosalveti** [comment]
Missing [FIO] tag on patch subject, otherwise LGTM

**Comment by igoropaniuk** [comment]
@ricardosalveti added tag

**Pull request merged**
